docs/reference/ccr/managing.asciidoc

```diff
@@ -0,0 +1,171 @@
+[role="xpack"]
+[testenv="platinum"]
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT /follower_index/_ccr/follow?wait_for_active_shards=1
+{
+  "remote_cluster" : "remote_cluster",
+  "leader_index" : "leader_index"
+}
+--------------------------------------------------
+// TESTSETUP
+// TEST[setup:remote_cluster_and_leader_index]
+
+[source,console]
+--------------------------------------------------
+POST /follower_index/_ccr/pause_follow
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+[[ccr-managing]]
+=== Manage {ccr}
+Use the following information to manage {ccr} tasks, such as inspecting
+replication progress, pausing and resuming replication, recreating a follower
+index, and terminating replication.
+
+[[ccr-inspect-progress]]
+==== Inspect replication statistics
+You can inspect the progress of replication for a follower index and view
+detailed shard statistics.
+
+. Open a browser to access {kib} on your _local_ cluster.
+. From the {kib} menu, click *Management -> Stack Management*.
+. Under *Data*, select *Cross Cluster Replication*.
+  . Choose the *Follower indices* tab.
+  . Select the name of the follower index you want to view replication details
+  for.
+
+The slide-out panel shows settings and replication statistics for the follower
+index, including read and write that are managed by the follower shard.
+
+To view more detailed statistics, click *View in Index Management*, and
+then select the name of the follower index on the *Index Management* page. Choose the *Stats* tab to view detailed statistics at the shard level.
+
+[%collapsible]
+.Use the API
+====
+Use the <<ccr-get-follow-stats,get follower stats API>> to inspect replication
+progress at the shard level. This API provides insight into the read and writes
+managed by the follower shard. The API also reports read exceptions that can be
+retried and fatal exceptions that require user intervention.
+====
+
+[[ccr-pause-replication]]
+==== Pause and resume replication
+Pause and resume replication of follower indices. When resuming, you can choose
```

Comment:
This should be clarified...it's not the follower indices that are replicated. :-)